<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-N9QFJ52" name="testRuns.Run_All" tests="2" failures="1" timestamp="02 May 2022 07:03:52 GMT" time="66.340" errors="0">
  <testcase name="runScenario" time="23.795" classname="testRuns.Run_All"/>
  <testcase name="runScenario" time="42.545" classname="testRuns.Run_All">
    <failure type="java.lang.AssertionError" message="expected [My Store1] but found [My Store]">
      <![CDATA[java.lang.AssertionError: expected [My Store1] but found [My Store]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at commonMethods.SpecificToSelenium.windowTitle(SpecificToSelenium.java:10)
at stepDefinations.Page1.user_is_on_page(Page1.java:12)
at âœ½.user is on "My Store1" page(file:///E:/workspace/Log_Cucumber/src/test/resources/features/page1/feature1.feature:4)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- testRuns.Run_All -->
